### DISCUSSION TITLE
Unable to rename collections

### DISCUSSION THREAD

**Discussion by kyledenlinger** [knowledgefutures/pubpub#2061] [Support] [answered]
Not sure if this is a hiccup or a larger issue. Currently it seems I'm unable to rename collections from the collection settings screen. It doesn't seem to be an issue with what I put into the title field, because I can create a new collection with the intended name without problem.

**Comment by kyledenlinger** [comment, marked as answer]
Ah, nevermind. Found [the previous posts describing the problem and workarounds](https://github.com/pubpub/pubpub/discussions/1420). Bad forum user, bad!

**Reply by gabestein** [reply]
Glad to hear it, and sorry about the duplicate response. Ships passing in the night. Anyhow, yeah...sorry about that. It's really bad ui. We're working on a larger overhaul to settings, but may try to fix it in the meantime because yeah it's bad.
